libs/components/toast/src/lib/modules/toast/types/toast-config.ts

```diff
@@ -13,7 +13,7 @@ export interface SkyToastConfig {
   type?: SkyToastType;
 
   /**
-   * Indicates whether to automatically close the toast.
+   * Whether to automatically close the toast.
```

Comment:
```suggestion
   * Whether to automatically close the toast. Only close toasts
   * automatically if users can access the messages after the toasts close.
```
I think this config property should include the same "only close toasts" caveat that we have in the description for the component property, especially given that the description for the component property doesn't currently show up in the live site.